meta-facebook/yv35-hd/src/platform/plat_class.c

```diff
@@ -296,5 +306,44 @@ void init_platform_config()
 		}
 	}
 
+	/* Set HSC OCP */
+	uint8_t ilim_adjust = ILIM_ADJUST_DEFAULT;
+	if (_2ou_status.present) {
+		if (hsc_module == HSC_MODULE_ADM1278) {
+			gpio_set(HSC_OCP_GPIO1_R, GPIO_LOW);
+			gpio_set(HSC_OCP_GPIO2_R, GPIO_LOW);
+			gpio_set(HSC_OCP_GPIO3_R, GPIO_HIGH);
+		} else if (hsc_module == HSC_MODULE_LTC4282) {
+			ilim_adjust = ILIM_ADJUST_2OU_EXP;
+		}
```

Comment:
Please add else clause to catch invalid hsc_module values.

Reply:
An else statement will be added to handle all conditions.